### PR TITLE
OrgUsers: Refactor change `LastSeenAtAge` from '10 years' to 'Never'

### PR DESCRIPTION
**Pull request by eleijonmarck**
**What is this feature?**
All users table shows `Never` when a user has never logged in. In the user organization tab we show `10 years` from previous implementation. This changes to `Never` in the org tab.

image

**Why do we need this feature?**
Aligning the all users table to the org users table.

### CODE REVIEW

File: public/app/features/admin/Users/OrgUsersTable.tsx [hunk 1]
```diff
@@ -107,7 +108,9 @@ export const OrgUsersTable = ({
       {
         id: 'lastSeenAtAge',
         header: 'Last active',
-        cell: ({ cell: { value } }: Cell<'lastSeenAtAge'>) => value,
+        cell: ({ cell: { value } }: Cell<'lastSeenAtAge'>) => {
+          return <>{value && <>{value === '10 years' ? <Text color={'disabled'}>Never</Text> : value}</>}</>;
```
Comment (mgyongyosi) [inline public/app/features/admin/Users/OrgUsersTable.tsx:112]:
```suggestion
          return <>{value && value === '10 years' ? <Text color={'disabled'}>Never</Text> : value}</>;
```

Comment (eleijonmarck) [reply public/app/features/admin/Users/OrgUsersTable.tsx]: changed in the `OrgUsersTable.tsx` and the `UsersTable.tsx`

**Review by Jguer — APPROVED** [review]

File: public/app/features/admin/Users/OrgUsersTable.tsx (hunk 1 above)
Comment (mgyongyosi) [reply public/app/features/admin/Users/OrgUsersTable.tsx]: @eleijonmarck Let me correct myself, the change that I recommended changes the result if `value === undefined`, so your first solution was right, could you please revert this change and the other one? Sorry 😇

Comment (eleijonmarck) [reply public/app/features/admin/Users/OrgUsersTable.tsx]: @mgyongyosi i tested out the changes locally 🤔 i think. let me first try locally again

**Pull request merged**